docusaurus/docs/ruby/reference.md

```diff
@@ -202,17 +202,6 @@ Available:
 - `development` sets `KNAPSACK_PRO_ENDPOINT` to `http://api.knapsackpro.test:3000`
 - `test` sets `KNAPSACK_PRO_ENDPOINT` to `https://api-staging.knapsackpro.com`
 
-## `KNAPSACK_PRO_MODIFY_DEFAULT_RSPEC_FORMATTERS` (RSpec)
```

Comment:
Should we keep it and add at the top something like:

```
This was removed in Knapsack Pro version 7.0 as it's not needed anymore.
```

We could link "version 7.0" to the changelog.